File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/location/SimpleLocationListener.java
package ch.rmy.android.statusbar_tacho.location;

import android.location.Location;
import android.location.LocationListener;
import android.os.Bundle;

public abstract class SimpleLocationListener implements LocationListener {

    @Override
    public void onLocationChanged(Location location) {

    }

    @Override
    public void onStatusChanged(String provider, int status, Bundle extras) {

    }

    @Override
    public void onProviderEnabled(String provider) {

    }

    @Override
    public void onProviderDisabled(String provider) {

    }

}

File: StatusBarSpeedometer/app/src/main/kotlin/ch/rmy/android/statusbar_tacho/location/SimpleLocationListener.kt
package ch.rmy.android.statusbar_tacho.location

import android.location.Location
import android.location.LocationListener
import android.os.Bundle

abstract class SimpleLocationListener : LocationListener {

    override fun onLocationChanged(location: Location) {

    }

    @Deprecated("Deprecated in Java")
    override fun onStatusChanged(provider: String, status: Int, extras: Bundle) {

    }

    override fun onProviderEnabled(provider: String) {

    }

    override fun onProviderDisabled(provider: String) {

    }

}
